docs/turbo-native-modules-ios.md

```diff
@@ -111,14 +109,56 @@ RCT_EXPORT_MODULE(NativeLocalStorage)
   }
 }
 
++ (NSString *)moduleName
+{
+  return @"NativeLocalStorage";
+}
+
 @end
 ```
 
 Important things to note:
 
-- `RCT_EXPORT_MODULE` exports and registers the module using the identifier we'll use to access it in the JavaScript environment: `NativeLocalStorage`. See these [docs](./legacy/native-modules-ios#module-name) for more details.
 - You can use Xcode to jump to the Codegen `@protocol NativeLocalStorageSpec`. You can also use Xcode to generate stubs for you.
 
+## Register the Native Module in your app
+
+The last step consist in updating the `package.json` to tell React Native about the link between the JS specs of the Native Module and the concrete implementation of those specs in native code.
+
+Modify the `package.json` as it follows:
+
+```json title="package.json"
```

Comment:
same as above

Reply:
Same as above: `//highlight-add-start` and `//highlight-add-end` will show the diff